doc/sphinx/using/libraries/writing.rst

```diff
@@ -57,7 +59,7 @@ deprecated compatibility alias using :cmd:`Notation (abbreviation)`
 
    .. coqtop:: all abort warn
 
-      #[deprecated(since="0.9", note="Use idtac instead.")]
+      #[deprecated(since="mylib-0.9", note="Use idtac instead.")]
```

Comment:
```suggestion
      #[deprecated(since="mylib 0.9", note="Use idtac instead.")]
```
so that it appears as such in the warning message (the dash will be automatically added for the warning name). At least that's what we do in MathComp.